buildSrc/src/main/kotlin/com/virtuslab/gitmachete/buildsrc/IntellijPlugin.kt

```diff
@@ -42,36 +42,22 @@ fun Project.configureIntellijPlugin() {
   configure<ChangelogPluginExtension> {
     val PROSPECTIVE_RELEASE_VERSION: String by extra
     version.set("v$PROSPECTIVE_RELEASE_VERSION")
-    unreleasedTerm.set("v$PROSPECTIVE_RELEASE_VERSION")
     headerParserRegex.set(Regex("""v\d+\.\d+\.\d+"""))
     path.set("${project.projectDir}/CHANGE-NOTES.md")
   }
 
   val changelog = extensions.getByType(ChangelogPluginExtension::class.java)
 
-  val verifyVersionTask = tasks.register("verifyChangeLogVersion") {
-    doLast {
-      val prospectiveVersionSection = changelog.get(changelog.version.get())
-      val latestVersionSection = changelog.getLatest()
-
-      if (prospectiveVersionSection.version != latestVersionSection.version) {
-        throw Exception(
-          "${prospectiveVersionSection.version} is not the latest in CHANGE-NOTES.md, " +
-            "update the file or change prospecitve version in version.gradle.kts"
-        )
-      }
-    }
-  }
-
   val verifyContentsTask = tasks.register("verifyChangeLogContents") {
```

Comment:
Whoops getting a false positive now on this PR

![image](https://user-images.githubusercontent.com/3383210/199113150-ed952230-63df-47b4-a761-4972bf359ac7.png)

Reply:
lemme see... 👁️

Reply:
I got it - now renderItem contains the version header e.g. `## v3.2.0` - fixed by dropping the first element